backend/src/main/java/com/salat/briene/exceptions/ArticleNotFoundException.java

```diff
@@ -3,17 +3,17 @@
 import java.io.IOException;
 
 public class ArticleNotFoundException extends IOException {
```

Comment:
We have a lot of checked exceptions and also a lot of propagations. For me, IOException is more system-level and shouldn't be triggered by user actions. We can extend from RuntimeException because we handling exceptions in ControllerAdvices.